.github/actions/clp-core-run-on-image/action.yaml

```diff
@@ -1,5 +1,5 @@
-name: "clp-core-build"
-description: "Builds CLP-core in the specified container"
+name: "clp-core-run-on-image"
```

Comment:
Actually I meant `run-on-image` by itself, without the prefix, for two reasons:

1. This seems like more of a general container image, not specific to clp-core.
2. The "clp-core-" prefix was only ever added since we wanted "core-" as the prefix but that sounds ambiguous by itself.